pkg/pool-weighted/test/WeightedPool.test.ts

```diff
@@ -62,7 +62,7 @@ describe('WeightedPool', function () {
 
         // Perform a recovery mode exit. This will reduce the invariant but this isn't tracked due to recovery mode.
         const preExitInvariant = await pool.getLastPostJoinExitInvariant();
-        await pool.recoveryModeExit({ from: lp, bptIn: fp(100) });
+        await pool.recoveryModeExit({ from: lp, bptIn: fp(10) });
```

Comment:
Why this change?

Reply:
The tests were failing due to `lp` not having enough BPT

Reply:
But why did the tests start failing? Did we change anything else?

Reply:
I've reverted the change and it seems to work again. I can't reproduce locally.